src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,178 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<doc>
+  <members>
+
+    <member name="TextTransform">
+      <summary>
+        A transform that turns a collection of text documents into numerical feature vectors.
+        The feature vectors are normalized counts of (word and/or character) ngrams in a given tokenized text.
+      </summary>
+      <remarks>
+        The TextFeaturizer transform gives user one-stop solution for doing:
+        <list type="bullet">
+          <item><description>Language Detection</description></item>
+          <item><description>Tokenzation​</description></item>
+          <item><description>Text normalization</description></item>
+          <item><description>Predefined and custom stopwords removal.</description></item>
+          <item><description>Word-based or character-based Ngram and SkipGram extraction.​</description></item>
+          <item><description>TF, IDF or TF-IDF.</description></item>
+          <item><description>L-p vector normalization.​</description></item>
+        </list>
+        The TextFeaturizer will show the transformed text, after being applied.
+        It converts a collection of text columns to a matrix of token  ngrams/skip-grams counts.
+        Features are made of (word/character) n-grams/skip-grams​ and the number of features are equal to the vocabulary size found by analyzing the data.
+      </remarks>
+    </member>
+    <example name="TextTransform">
+      <example>
+        <code language="csharp">
+          pipeline.Add(new TextFeaturizer(&quot;Features&quot;, &quot;SentimentText&quot;)
+          {
+          KeepDiacritics = false,
+          KeepPunctuations = false,
+          TextCase = TextNormalizerTransformCaseNormalizationMode.Lower,
+          OutputTokens = true,
+          StopWordsRemover = new PredefinedStopWordsRemover(),
+          VectorNormalizer = TextTransformTextNormKind.L2,
+          CharFeatureExtractor = new NGramNgramExtractor() { NgramLength = 3, AllLengths = false },
+          WordFeatureExtractor = new NGramNgramExtractor() { NgramLength = 2, AllLengths = true }
+          });
+        </code>
+      </example>
+    </example>
+
+    <member name="WordTokenizer">
+      <summary>
```

Comment:
>summary [](start = 7, length = 7)

again, this would make a good first paragraph in remarks, and the summary could be
```
Splits the text into words using the separator character(s).
```

Reply:
I don't see the changes done, only one sentence is gone.

---
In reply to: [203203181](https://github.com/dotnet/machinelearning/pull/529#discussion_r203203181) [](ancestors = 203203181)

Reply:
thanks for catching. 

---
In reply to: [203513343](https://github.com/dotnet/machinelearning/pull/529#discussion_r203513343) [](ancestors = 203513343,203203181)